### ISSUE TITLE
Web Bluetooth Notifications NotSupportedError

### ISSUE THREAD

**Issue by chipimix** [GoogleChrome/samples#601]
Hello!
I am trying to connect a [TI SensorTag CC2650](http://processors.wiki.ti.com/images/a/a8/BLE_SensorTag_GATT_Server.pdf) to Web Bluetooth in a Chrome browsers running on Win10.
I have used [this demo](https://googlechrome.github.io/samples/web-bluetooth/characteristic-properties.html) successfully with the following UUID parameters (Humidity Sensor):
Bluetooth Service: f000aa20-0451-4000-b000-000000000000
Bluetooth Characteristic: f000aa21-0451-4000-b000-000000000000
It outputs: 
```
> Characteristic UUID:  f000aa21-0451-4000-b000-000000000000
> Broadcast:            false
> Read:                 true
> Write w/o response:   false
> Write:                false
> Notify:               true
> Indicate:             false
> Signed Write:         false
> Queued Write:         false
> Writable Auxiliaries: false
```
Everything in its right place so far :)
Here's what I don't understand:
In the [notification example sample](https://googlechrome.github.io/samples/web-bluetooth/notifications.html), using the same parameters as before i get the following Error:
`Argh! NotSupportedError: GATT operation failed for unknown reason.
`

Why is this happening? :(

**Comment by chipimix** [comment]
TLDR: Is it possible to fetch, say, the heartrate value from a mi band2 using web bluetooth on windows? Which example should I explore further?
Thanks in advance!!
